src/tests/multihost/alltests/test_kcm.py

```diff
@@ -52,3 +52,36 @@ def test_client_timeout(self, multihost, backupsssdconf):
                                                    " /var/log/sssd/"
                                                    "sssd_kcm.log")
         assert 'Terminated client' in grep_cmd.stdout_text
+
+    def test_kcm_check_socket_path(self, multihost, enable_kcm):
+        """
+        @Title: kcm: Test socket path when sssd-kcm is activated by systemd
+        #https://github.com/SSSD/sssd/issues/5406
+        """
+        # Start from a known-good state after removing log file and adding a
+        # new socket path
+        multihost.master[0].service_sssd('stop')
+        self._stop_kcm(multihost)
+        self._remove_kcm_log_file(multihost)
+        server = sssdTools(multihost.master[0])
+        server.backup_sssd_conf()
+        socket_path = "/some_path/kcm.socket"
+        domain_section = "kcm"
+        sssd_params = {'socket_path': '%s' % (socket_path)}
+        server.sssd_conf(domain_section, sssd_params)
+        multihost.master[0].service_sssd('start')
+        self._start_kcm(multihost)
```

Comment:
Similar to stop
multihost.client[0].run_command("systemctl start sssd-kcm")

or function #5512